platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -230,6 +228,8 @@ void updateSepTransaction(PlatformTransactionData patch, JdbcSepTransaction txn)
         }
         break;
       case "31":
+        txnUpdated = updateField(patch, txn, "transferReceivedAt", txnUpdated);
+
```

Comment:
Do we not want to update this field in SEP-24 withdrawals?

Reply:
It's not returned in GET response of SEP-24, so I don't think it makes sense to update it at all

Reply:
It should be in the Platform API's transaction response though